cwf/gateway/integ_tests/multi_sessioneproxy_test.go

```diff
@@ -147,26 +146,48 @@ func getPCRFandOCSnamePerInstance(instanceId int) (pcrfName string, ocsName stri
 	return
 }
 
-// TODO:
-//  * Support for multiple UEs (depends on UEsim service)
-//  * Check OCS credit has been reported (right now sessiond sends CCR after accounts
-//    are deleted from OCS and PCRF)
+
 // TestMultiSessionProxyMonitorAndUsageReportEnforcement is an experimental
 // test to try multiple OCS and PCRF servers. Currenty it only supports 1 UE
 // - Create one UE and add monitoring key and credit
 // - Attach UE, tranfer data, detach
 // - Check that the Monitored data by the PCRF instance is good
+// TODO:
+//  * Support for multiple UEs (depends on UEsim service)
+//  * Check OCS credit has been reported (right now sessiond sends CCR after accounts
+//    are deleted from OCS and PCRF)
 func TestMultiSessionProxyMonitorAndUsageReportEnforcement(t *testing.T) {
-	fmt.Println("\nRunning TestMultiSessionProxyUsageReportEnforcement...")
+	fmt.Println("\nRunning TestMultiSessionProxyMonitorAndUsageReportEnforcement...")
+	tr := NewTestRunnerWithTwoPCRFandOCS(t)
+
+	// Overwrite the mconfig to enable multiproxy with OCS2 and PCRF2
+	err := tr.OverwriteMConfig("./gateway.mconfig.multi_session_proxy", "session_proxy")
+	assert.NoError(t, err)
+	assert.NoError(t, tr.RestartService("ocs"))
+	assert.NoError(t, tr.RestartService("pcrf"))
+	assert.NoError(t, tr.RestartService("ocs2"))
+	assert.NoError(t, tr.RestartService("pcrf2"))
+	assert.NoError(t, tr.RestartService("ingress"))
+	assert.NoError(t, tr.RestartService("session_proxy"))
```

Comment:
Does this block until the services are started? I forget. I remember at some point I had an issue where the tests ran before something was not ready... :/

Reply:
Per what i understand from the ContainerRestart function from go docker, it does wait for the container to be restarted and be healthy (it doensn't mean that the service itself needs for something else to wait)

But I can add wait before session_proxy to give some kind of assurance that the core will be ready once it restarts